doc/release-notes/2.2.md

```diff
@@ -1,10 +1,65 @@
+    modelBuilder.Entity<Blog>()
+        .ForNpgsqlHasIndex(b => b.Id)
+        .ForNpgsqlInclude(b => b.Name);
+}
+```
+
+This will create an index for searching on `Id`, but containing also the column `Name`, so that reading the latter will not involve accessing the table. [See the documentation for more details](../modeling/indices.md).
+
+Thanks to @khellang for contributing this!
+
+### PostgreSQL user-defined ranges
+
+The provider already supported [PostgreSQL range types](https://www.postgresql.org/docs/current/rangetypes.html), but prior to 2.2 that support was limited to the built-in range types which come with PostgreSQL. [#329](https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/issues/329) extends that support to range types which you define:
+
+```c#
+protected override void OnConfiguring(DbContextOptionsBuilder builder) {
+    builder.UseNpgsql("...", b => b.MapRange("floatrange", typeof(float)));
+}
+
+
+protected override void OnModelCreating(ModelBuilder builder) {
+    builder.ForNpgsqlHasRange("floatrange", "real");
+}
+```
+
+This will make the provider create a PostgreSQL range called `floatrange`, over the PostgreSQL type `real`. Any property with type `NpgsqlRange<float>` will be seamlessly mapped to it.
+
+[See the documentation for more details](../mapping/range.md).
+
+### Type-constrained PostGIS columns
+
+The NetTopologySuite previously created PostGIS columns of type `GEOMETRY` (or `GEOGRAPHY`) in the database. With ([#719](https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/issues/719)), a property of type `Point` will be instead mapped to columns of type `GEOMETRY (POINT)`.
+
+### Seeding for Npgsql-specific types
+
+When using some Npgsql-specific types, it wasn't possible to seed values for those types. With EF Core support for seeding any type, ([#667](https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/issues/667)) allows seeding values for network, bit and range types (more are coming).
+
+### PostgreSQL index operator classes
+
+PostgreSQL allows you to specify [operator classes on your indices](https://www.postgresql.org/docs/current/indexes-opclass.html), to allow tweaking how the index should work. ([#481](https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/issues/481)) adds support for managing these. [See the documentation for more details](../modeling/indices.md).
+
+Thanks to @khelland for contributing this!
+
+### Other features
 
-## Breaking changes
+* Various issues with enum and range types were fixed, including upper/lower case, quoting and schema management.
+* Many new SQL translations were added, so more of your LINQ expressions can run in the database. We'll be working on our documentation to make these more discoverable.
 
+The full list of issues for this release is [available here](https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/issues?page=2&q=milestone%3A2.2.0+is%3Aclosed&utf8=%E2%9C%93).
```

Comment:
Is that link to page 2 of the issues list? (Reviewing on mobile, so I can't click it at the moment.)

Reply:
Great catch, thanks!